### PR TITLE
Hotfix/toaster not closing

### PR DESCRIPTION
**Pull request by virtuoso98**
fixed toaster not closing sometimes when bulk analysis is performed

### CODE REVIEW

**Comment by virtuoso98** [comment]
My bad, I forked this branch off the tool-tip branch. Would recommend you review that branch first before reviewing this.

**Review by beesaycheese — APPROVED** [review]
LGTM

File: src/app/src/components/annotations/annotator.tsx [hunk 1]
```diff
@@ -649,8 +652,9 @@ export default class Annotator extends Component<
       // eslint-disable-next-line no-await-in-loop
       await this.getInference(asset, true);
       if (
-        this.state.inferenceOptions.bulkAnalysisStatus === "both" ||
-        this.state.inferenceOptions.bulkAnalysisStatus === asset.type
+        (this.state.inferenceOptions.bulkAnalysisStatus === "both" ||
+          this.state.inferenceOptions.bulkAnalysisStatus === asset.type) &&
```
Comment (virtuoso98) [inline src/app/src/components/annotations/annotator.tsx]: I think this becomes rather unnecessary given that you're already filtering on the top level

**Review by virtuoso98 — COMMENTED** [review]
One small comment on this PR - think that lines 655 and 656 are unnecessary now that you already filtered them before running the for-loop

File: src/app/src/components/annotations/annotator.tsx (hunk 1 above)
Comment (beesaycheese) [reply src/app/src/components/annotations/annotator.tsx]: got it!

**Pull request merged**